### ISSUE TITLE
eb logs --all errors with 'ascii' codec can't encode character '\u25b8'

### ISSUE THREAD

**Issue by alexandargyurov** [aws/aws-elastic-beanstalk-cli#15]
### Description
Our application is running Elixir and we have the following character in our output: `▸`

When we run `eb logs  --all`, the commands fails with:

`ERROR: UnicodeEncodeError - 'ascii' codec can't encode character '\u25b8' in position 524958: ordinal not in range(128)`

### Steps to reproduce

Have logs that contain unicode character `\u25b8`.

### Observed result

```2020-02-28 10:57:49,996 (INFO) eb : Traceback (most recent call last):
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/ebcli/core/ebrun.py", line 62, in run_app
    app.run()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/cement/core/foundation.py", line 797, in run
    return_val = self.controller._dispatch()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/cement/core/controller.py", line 472, in _dispatch
    return func()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/cement/core/controller.py", line 478, in _dispatch
    return func()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/ebcli/core/abstractcontroller.py", line 92, in default
    self.do_command()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/ebcli/controllers/logs.py", line 65, in do_command
    self.__get_logs()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/ebcli/controllers/logs.py", line 97, in __get_logs
    self.__retrieve_cloudwatch_instance_logs(info_type, should_zip_logs)
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/ebcli/controllers/logs.py", line 121, in __retrieve_cloudwatch_instance_logs
    logsops.retrieve_cloudwatch_instance_logs(
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/ebcli/operations/logsops.py", line 506, in retrieve_cloudwatch_instance_logs
    retrieve_cloudwatch_logs(
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/ebcli/operations/logsops.py", line 564, in retrieve_cloudwatch_logs
    _write_full_logs_to_file(full_logs, logs_location, log_stream)
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.17.1/libexec/lib/python3.8/site-packages/ebcli/operations/logsops.py", line 966, in _write_full_logs_to_file
    log_file.write(full_logs)
UnicodeEncodeError: 'ascii' codec can't encode character '\u25b8' in position 505357: ordinal not in range(128)

2020-02-28 10:57:49,997 (INFO) eb : UnicodeEncodeError - 'ascii' codec can't encode character '\u25b8' in position 505357: ordinal not in range(128)
```

### Expected result

Downloads logs

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: Mac OS & Linux

**Comment by halcarleton** [comment]
Hi @alexandargyurov,

Thanks for reporting this. I have not been able to reproduce this. I have tried several approaches with no luck. If possible could you upload a log file that is causing this issue for you(be sure to sanitize any sensitive data)?

What platform version are you using?

**Comment by halcarleton** [comment]
Ok, I was finally able to reproduce the issue. The appears to be a locale issue. I'm going to dig into the cause and will post updates here.

**Comment by halcarleton** [comment]
This issue should now be resolved in [v3.18.1](https://pypi.org/project/awsebcli/3.18.1/)
